Mastodon/Scene/Onboarding/Welcome/View/Pages/WelcomeContentPage.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  WelcomeContentPage.swift
+//  Mastodon
+//
+//  Created by Nathan Mattes on 26.11.22.
+//
+
+import UIKit
+import MastodonLocalization
+
+enum WelcomeContentPage: CaseIterable {
```

Comment:
As this enum is, at least currently, only used to switch over different content with the same layout, I'm wondering if we should make this more generic e.g. `pageOne`, `pageTwo`, etc. then we'd only need to update the i18n values later on, if we'd like to change the onboarding content.